Makefile

```diff
@@ -314,13 +313,6 @@ else
 	bash ./operations/helm/scripts/rebuild-tests.sh
 endif
 
-generate-manifests:
-ifeq ($(USE_CONTAINER),1)
-	$(RERUN_IN_CONTAINER)
-else
-	cd production/kubernetes/build && bash build.sh
-endif
```

Comment:
We have no static manifests to generate anymore, so this target has been removed.